website/blog/2022/04/29/interview-juzhiyuan-apache-member.md

```diff
@@ -0,0 +1,340 @@
+---
+title: "Interview with JU Zhiyuan: Becoming One of the 918 ASF Members, I Felt Thrilled and Proud"
+authors:
+  - name: "Estelle Rao"
+    title: "Author"
+    url: "https://github.com/EstelleRao"
+    image_url: "https://github.com/EstelleRao.png"
+keywords:
+- APISIX
+- Apache APISIX
+- API Gateway
+- ASF Member
+- Apache Member
+- Open Source
+- Remote Work
+- Full-Stack Web Developer
+description: In this interview, we invited JU Zhiyuan, a PMC member of Apache APISIX, to share his feeling about becoming an ASF member, his experience with open source, and his self-learning journey in technology.
+tags: [Interview]
+---
+
+>JU Zhiyuan, a PPMC member of Apache APISIX, was elected to be one of the 918 ASF Members in March 2022. In this interview, Zhiyuan shared his experience with open source and his self-learning journey in technology.
+
+<!--truncate-->
+
+**JU Zhiyuan (GitHub ID: juzhiyuan), born in 1997, became an Apache APISIX PPMC member in October 2019 and was elected as  
+an ASF Member in March 2022. He is an active advocate for open source as an ApacheCon Speaker, OSPP Mentor, and GSoC Mentor  
+and was one of the core organizers of freeCodeCamp China before 2019, spreading the word of open source and giving help and support for open-source newbies. Currently, he leads a global team in API7.ai, an open-source software company.**
```

Comment:
If this is a paragraph, please do not split it into multiple lines. Please check the full text.